SU-HSDO/suhumsci
# HSD8-468 Global grunt sass compiler

# MAYBE READY FOR REVIEW

# Summary
- Global sass compiler for all humsci code

# Needed By (Date)
- no rush.

# Urgency
- the lowest possible

# Steps to Test

1. checkout this branch
1. `npm install`
1. `grunt watch`
1. edit some humsci scss files
1. validate compiles as expected.


# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
